README.md

```diff
@@ -40,6 +40,12 @@ We are more interested in seeing your thought process and approach to solving th
 We will expect to see the following items in your Github pull request:
 
 - Your Python code for data fetcher and transformer.
+  - In the transformer, please create data models for:
```

Comment:
Personally, I would make this bit of the instructions less clear, i.e.:
* Make a new bullet at the same level as "- Your Python code for data fetcher and transformer."
* And just ask the candidate to "please create data models for", i.e. do not mention that this is expected to be in the transformer

Reply:
This is a good idea. Let's have it for v2. We still need to calibrate the challenge with the candidate experience and I need to be a bit cautious.